src/data_utils.py

```diff
@@ -0,0 +1,23 @@
+import csv
+
+def get_easy_ids(file_path):
+    """
+    Reads a CSV file and returns a list of IDs where the 'difficulty' column is 'easy'.
+    
+    Parameters:
+        file_path (str): The path to the CSV file.
+
+    Returns:
+        list: A list of IDs where the difficulty is 'easy'.
+    """
+    easy_ids = []
+    try:
+        with open(file_path, 'r') as csv_file:
+            reader = csv.DictReader(csv_file)
+            for row in reader:
```

Comment:
Just noticed this: you can avoid using a for loop here if you read the csv into a pandas dataframe (it should fit in memory) and use something like `df['difficulty'].str.lower().str.strip() == 'easy'`.